components/event-bus/internal/knative/util/kn-eventing.go

```diff
@@ -58,13 +59,15 @@ if err := k.CreateSubscription("my-sub", namespace, channelName, &uri); err != n
 return
 */
 
+const hyphen = "-"
```

Comment:
Changing the value would void the name, which defies the purpose of the constant. Maybe rename it to something that describes a purpose, not a value (e.g. `replacementChar`, `delimiter`, ...)

Reply:
changed to *generatedNameSuffix*